app/assets/stylesheets/pages/_challenges-index.scss

```diff
@@ -0,0 +1,54 @@
+.challenges-container {
+  max-width: 700px;
+}
+
+.challenge-list {
+  position: relative;
+  min-height: 90px;
+  background-color: #EAEAEA;
+  border-radius: 5px;
+  border: 2px solid $gray;
+  margin: 5px 0px;
+}
+
+.challenge-name {
+  position: absolute;
+  height: 100%;
+  padding: 10px;
+  top: 0px;
+  width: 65vw;
+}
+
+.challenge-activate {
+  height: 100%;
+  position: absolute;
+  background-color: $blue;
+  color: $white;
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+  padding: 20px 10px;
+  right: 0;
+}
+
+.challenge-title {
```

Comment:
We should make a clear distinction in class names for challenge cards (show page) and challenges cards (index page)